frontend/src/components/common/NavBar.tsx

```diff
@@ -68,121 +80,112 @@ const NavBar = (): JSX.Element => {
     }
   };
 
+  const location = useLocation();
+  const isAdminPage =
+    location.pathname.substring(0, Routes.ADMIN_PREFIX_LENGTH) ===
+      Routes.CAMPS_PAGE || location.pathname === Routes.HOME_PAGE;
+
   return (
     <Container maxWidth="100vw">
-      <Flex
-        direction="row"
-        justifyContent="space-between"
-        height="40px"
-        marginLeft="80px"
-        marginRight="80px"
-        marginTop="14px"
-        marginBottom="14px"
-      >
-        {authenticatedUser ? (
-          <>
-            <div
-              style={{
-                display: "flex",
-                flex: "1",
-                minWidth: "40px",
-              }}
-            >
-              <Image
-                src={FONIcon}
-                alt="FON icon"
-                display="inline"
-                width="40px"
-                height="40px"
-              />
-            </div>
-            <Tabs
-              variant="unstyled"
-              onChange={navigate}
-              defaultIndex={getIndex()}
-              width="650px"
-              alignContent="center"
-              isFitted
-            >
-              <TabList>
-                <Tab
-                  id="Camps"
-                  _selected={{
-                    color: "primary.green.100",
-                    fontWeight: FontWeights.SEMIBOLD,
-                  }}
-                >
-                  <FontAwesomeIcon icon={faCampground} />
-                  &nbsp; Camps
-                </Tab>
-                {authenticatedUser.role === Role.ADMIN && (
-                  <>
-                    <Tab
-                      id="Global Forms"
-                      _selected={{
-                        color: "primary.green.100",
-                        fontWeight: FontWeights.SEMIBOLD,
+      {authenticatedUser && isAdminPage && (
```

Comment:
I'd prefer to have this condition in `App.tsx`, that way we're not rendering a potentially empty container, but wasn't possible since `useLocation()` wasn't available there (I believe it needs to be in `<Switch>`

Reply:
Yep, from docs (https://v5.reactrouter.com/web/api/Hooks/uselocation):

> The useLocation hook returns the [location](https://v5.reactrouter.com/web/api/location) object that represents the current URL. You can think about it like a useState that returns a new location whenever the URL changes.

Should reload the comment as needed